guardian/apps.py

```diff
@@ -7,5 +7,8 @@ class GuardianConfig(AppConfig):
     name = 'guardian'
 
     def ready(self):
+        # Must patch Group here since generic
+        # group permission model is definable
+        monkey_patch_group()
```

Comment:
I would think this should be conditional, same as the `monkey_patch_user` below... Yes, I know this wasn't the case before...

Reply:
Making `monkey_patch_group()` call dependent on `GUARDIAN_MONKEY_PATCH = True` breaks the use case of `GuardianUserMixin` since `add_obj_perm()` and `del_obj_perm()` would not be defined on `django.contrib.auth.model.Group`.

To clarify `GuardianUserMixin` use case:
- custom user model
- custom user model is dependent on guardian library
- `GUARDIAN_MONKEY_PATCH = False` 
- `GuardianUserMixin` is mixed in with the custom user model
For example:
```
class CustomUser(GuardianUserMixin, AbstractUser):
    # Custom added fields  etc ....
    pass
```

_TL;DR My current understanding, not always calling monkey_patch_group() will break guardian._

Reply:
Wouldn't there be the same breakage not calling `monkey_patch_user()`? Or is that different?

Reply:
No, when `monkey_user_patch()` is not called then custom user model must subclass `guardian.mixins.GuardianUserMixin`.

[Guardian Docs - Custom User Model](https://django-guardian.readthedocs.io/en/stable/userguide/custom-user-model.html#custom-user-model)